From 5aa83eb9f216581558142cee19e92f792c5ef603 Fri 26  8 23:16:47 2022
From: Błażej Szczygieł <spaz16@wp.pl>
Date: Fri, 26 Aug 2022 23:16:47 UTC
Subject: [PATCH] Fix compilation with older Qt

Move is protected by Q_DISABLE_COPY in this case, too.

diff --git a/src/qmplay2/StreamInfo.hpp b/src/qmplay2/StreamInfo.hpp
index 9264ed5a..823f9cbe 100644
--- a/src/qmplay2/StreamInfo.hpp
+++ b/src/qmplay2/StreamInfo.hpp
@@ -49,7 +49,7 @@ enum QMPlay2Tags
 class QMPLAY2SHAREDLIB_EXPORT StreamInfo
 {
     Q_DECLARE_TR_FUNCTIONS(StreamInfo)
-    Q_DISABLE_COPY_MOVE(StreamInfo)
+    Q_DISABLE_COPY(StreamInfo)
 
 public:
     static QMPlay2Tags getTag(const QString &tag);
